packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts

```diff
@@ -0,0 +1,8 @@
+import { useExperimentVariant } from "@/hooks/useExperimentVariant";
+
+type UsePromptToCreateContentExperimentReturnType = { eligible: boolean };
+
+export function usePromptToCreateContentExperiment(): UsePromptToCreateContentExperimentReturnType {
+  const variant = useExperimentVariant("slicemachine-prompt-to-create-content");
+  return { eligible: variant?.value === "on" };
```

Comment:
What happen if a user deactivated telemetry? It will not be eligible with that, right? I thought it was something we wanted, as written in the ticket.

Reply:
yes, I guess user without telemetry will not see that feature.
@xrutayisire which ticket are you referring to? 
I fas as I understand this feature will be subject to ab-test, is there any other way to do ab-testing that doesn't require telemetry?

Reply:
ah, ok
sorry, I haven't seen that ticket before
will add the fallback then!